services/celery/celery.py

```diff
@@ -3,7 +3,7 @@
 from settings_config.celery import app
 
 
-@app.task(name="check_withdrawal")
+@app.task(name="check_withdrawal", default_retry_delay=4 * 60)
```

Comment:
can you explain default_retry_delay ?

Reply:
Yup I forgot to discuss this with you actually it's for retry if any task is get fail it will retry in 
every 4 minutes